src/app/jobs/types.ts

```diff
@@ -39,45 +39,53 @@ export interface PathwayPredictionResult {
   id: number;
   host: string;
   model: DeCaF.Model;
-  manipulations: {
+  manipulations?: {
     direction: 'delta' | 'down' | 'up';
     value: string;
+    id: string;
   }[];
   knockouts: string[];
-  heterologous_pathway: string[];
+  heterologous_reactions: string[];
   fitness: number;
   yield: number;
   product: number;
   biomass: number;
+  exotic_cofactors;
```

Comment:
Is it possible to figure out the type for this? I guess it's a bit unfortunate that there aren't proper schema definitions in the [metabolic ninja api](https://api-staging.dd-decaf.eu/pathways-caffeine/).